src/schema/v2/partnerOfferToCollector.ts

```diff
@@ -0,0 +1,73 @@
+import { date } from "./fields/date"
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLBoolean,
+  GraphQLEnumType,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { IDFields, NodeInterface } from "./object_identification"
+import { priceDisplayText } from "lib/moneyHelpers"
+import { connectionWithCursorInfo } from "./fields/pagination"
+
+export const PartnerOfferToCollectorType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "PartnerOfferToCollector",
+  interfaces: [NodeInterface],
+  fields: () => ({
+    ...IDFields,
+    artworkId: {
+      type: GraphQLString,
+      resolve: ({ artwork_id }) => artwork_id,
+    },
+    createdAt: date(),
+    endAt: date(),
+    isActive: {
+      type: GraphQLBoolean,
+      resolve: ({ active }) => active,
+    },
+    isAvailable: {
+      type: GraphQLBoolean,
+      resolve: ({ available }) => available,
+    },
+    partnerId: {
+      type: GraphQLString,
+      resolve: ({ partner_id }) => partner_id,
+    },
+    priceWithDiscountMessage: {
+      type: GraphQLString,
+      resolve: ({ price_with_discount: price, price_currency: currency }) => {
+        if (!price || !currency) {
+          return null
+        }
+
+        const priceCents = price * 100
```

Comment:
We should avoid harding the major to minor unit conversion here, and instead, use the correct ratio based on  the currency. Here is an [example](https://github.com/artsy/metaphysics/blob/23fae702991340bfa4901b90224e6189da0912c0/src/schema/v2/artwork/index.ts#L1195-L1197) to look up with MP's own json file.